modules/core/src/main/forum.scala

```diff
@@ -9,32 +9,26 @@ import reactivemongo.api.bson.Macros.Annotations.Key
 
 sealed class BusForumPost
 object BusForumPost:
-  given busForumPostChannel: bus.WithChannel[BusForumPost] with
-    def channel = "forumPost"
+  given bus.WithChannel[BusForumPost] = bus.InChannel("forumPost")
```

Comment:
Still too much boilerplate for my taste.

The current code cost of adding a buss message is one LOC `case class MyMessage(...)`.
This PR in its current state triples it.

Reply:
it'll be much less boilerplate with my suggestion here: https://github.com/lichess-org/lila/pull/15170#discussion_r1584188855

It'll be one enum, and one case for each message.